common/src/main/resources/config.yml

```diff
@@ -34,6 +34,7 @@ hooks:
     enabled: true
   LuckPerms:
     enabled: true
+    optimize_permission_lookup: true
```

Comment:
Maybe enable_permission_cache would be a better name for the option?

Reply:
Done

Reply:
Renamed to use_permission_cache

Reply:
fields, methods also renamed